### DISCUSSION TITLE
Does a pyright badge exist?

### DISCUSSION THREAD

**Discussion by JanEricNitschke** [microsoft/pyright#5075] [General]
I mean something comparable to [![Checked with mypy](http://www.mypy-lang.org/static/mypy_badge.svg)](http://mypy-lang.org/) to show that one is using pyright to type check the repository?

**Comment by erictraut** [comment]
That's a good idea. I added one to the pyright repo and documented how to use it [here](https://microsoft.github.io/pyright/#/ci-integration?id=adding-pyright-badge-to-readmemd).

**Comment by JanEricNitschke** [comment]
Thanks a lot!
